ast/src/parsed/mod.rs

```diff
@@ -54,7 +54,7 @@ pub enum PilStatement<T> {
         Vec<PilStatement<T>>,
         Option<Expression<T>>,
     ),
-    FunctionCall(usize, String, Vec<Expression<T>>),
+    Expression(usize, Box<Expression<T>>),
```

Comment:
I don't think it needs to be boxed, see the `Namespace` variant of this enum.

Reply:
Ah right!